server/__main__.py

```diff
@@ -2,74 +2,72 @@
 import json
 
 import websockets
+from connections import Connections
```

Comment:
```suggestion
from .connections import Connections
```
Making this a relative import makes more sense in this case